packages/serverless/scripts/build-awslambda-layer.js

```diff
@@ -75,6 +74,7 @@ async function collectPackages(cwd, packages = {}) {
 }
 
 async function main() {
+  const baseDir = path.resolve(__dirname, '../../../')
```

Comment:
We need the layer to be built in the project root for the [sentry lambda-layer action](https://github.com/getsentry/action-build-aws-lambda-extension), right? 
I think it's fine for now but ultimately I'd prefer it to be built in the serverless package because it would be more coherent with our other builds. WDYT is this something we can do in the future?

Reply:
The GH action that takes this dist directory and adds the extension and then zips it expects a `/dist-serverless` in the root folder because it is used by Python and JS (and in the future maybe Ruby, Go, Java, ...)

So I guess for now it is ok, but I also get that you would like to have them in the package folder. Maybe we can figure out some common root folder where compiled stuff like this can go, that makes everyone happy. Or we could just put everything in a `tmp` folder so it is completely out of the way.

Reply:
On another thought the `/tmp/` folder is harder to implement to share code between github actions. 
So I think we just stick with how it is now.

I know it is not beautiful to polute the root dir of a project with a directory and a zip file. But the building of the lambda layer is by default not done when you run `yarn build` locally, so I guess it is ok?

Reply:
I think that's a fair point - let's keep it that way for now. All options considered I don't think that `tmp` makes it particularly better and it'll always be hard to specify one directory that fits the structure of multiple repos nicely. I was kinda thinking that maybe we could adjust the GH action to e.g. specify a custom directory but this is definitely something we can do in the future. Thanks for considering my input!